schema/feature_endpoint_mapping.json

```diff
@@ -264,15 +264,33 @@
     "dependent_on": []
   },
   "/api/*/s3/iam/users": {
-    "feature_name": "s3_iam_users",
+    "feature_name": "s3_iam_user",
```

Comment:
s3_iam_user -> iam_user_mgmt

Reply:
updated

Reply:
Please revert the changes to the original one.

Reply:
reverted